Configurations/10-main.conf

```diff
@@ -1624,13 +1640,12 @@ my %targets = (
     "darwin-common" => {
         inherit_from     => [ "BASE_unix" ],
         template         => 1,
-        cc               => "cc",
-        cflags           => picker(default => "",
-                                   debug   => "-g -O0",
+        CC               => "cc",
+        CFLAGS           => picker(debug   => "-g -O0",
                                    release => "-O3"),
+        LDFLAGS          => "-Wl,-search_paths_first",
```

Comment:
I'd say this is required. Without it it will end up trying to link with system libraries, that's not what we need, we want to link with libraries in the build tree.

Reply:
Thanks, will fix